tests/ipsec/before_test.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright 2017-2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary:  Basic preparation before any IPSec test
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+use Mojo::Base "ipsecbase", -signatures;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+use lockapi;
+
+sub run ($self) {
+    select_serial_terminal;
+
+    # disable packagekitd
+    quit_packagekit();
+    ensure_apparmor_disabled();
+
+    # Stop firewall
+    systemctl 'stop ' . $self->firewall;
+
+    set_hostname(get_var('HOSTNAME', 'susetest'));
+}
+
+sub ensure_apparmor_disabled () {
+    unless (systemctl "is-active apparmor", proceed_on_failure => 1) {    # 0 if active, unless to revert
+        systemctl "disable --now apparmor";
+        record_info "apparmor", "disabled";
+    }
+}
+
+sub test_flags ($self) {
+    return {fatal => 1, milestone => 1};
+}
+sub post_run_hook ($self) { }
+
```

Comment:
Why we need this module at all? Idea looks copied from HPC, which is old approach.

Reply:
Can be remove, will update it.